src/main/java/org/owasp/benchmark/score/parsers/AcunetixReader.java

```diff
@@ -0,0 +1,248 @@
+/**
+* OWASP Benchmark Project
+*
+* This file is part of the Open Web Application Security Project (OWASP)
+* Benchmark Project For details, please see
+* <a href="https://owasp.org/www-project-benchmark/">https://owasp.org/www-project-benchmark/</a>.
+*
+* The OWASP Benchmark is free software: you can redistribute it and/or modify it under the terms
+* of the GNU General Public License as published by the Free Software Foundation, version 2.
+*
+* The OWASP Benchmark is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+* even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details
+*
+* @author Dave Wichers
+* @created 2015
+*/
+
+package org.owasp.benchmark.score.parsers;
+
+import java.util.List;
+
+import org.owasp.benchmark.score.BenchmarkScore;
+import org.w3c.dom.Node;
+
+public class AcunetixReader extends Reader {
+	
+	public TestResults parse( Node root ) throws Exception {
+
+      if (root.getNodeName().equalsIgnoreCase("acunetix-360")) {
+
+	/* This is for the 2020 format that looks like:
+		 <acunetix-360 generated="28/02/2020 08:59 AM">
+			<target>
+				<scan-id>f55650d3326f40f30d92ab6e04af1794</scan-id>
+				<url>https://localhost:8443/benchmark/</url>
+				<initiated>27/02/2020 04:49 PM</initiated>
+				<duration>01:57:21.2094646</duration>
+			</target>
+			<vulnerabilities>
+				<vulnerability>
+					<LookupId>cf5644f1-31ee-4092-0e15-ab6e04af7f51</LookupId>
+					...
+	*/
+        TestResults tr = new TestResults( "Acunetix 360", true, TestResults.ToolType.DAST);
+        
+        Node target = getNamedChild( "target", root );
+        String duration = getNamedChild("duration", target ).getTextContent();
+        // duration format is: 01:57:21.2094646
+        tr.setTime( duration.substring(0, duration.lastIndexOf('.')));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

If the lastIndexOf() search on line 50 does not find the substring and returns -1, the call to substring() on line 50 will throw IndexOutOfBoundsException.